package.json

```diff
@@ -12,7 +12,8 @@
     "bootstrap": "lerna bootstrap",
     "bump": "lerna publish",
     "codecov": "lerna run codecov",
-    "check": "lerna run check"
+    "check": "lerna run check",
+    "docs": "typedoc --tsconfig  ./packages/opentelemetry-types/tsconfig.json"
```

Comment:
It feels wrong that the root would use a TS config from one of the packages. Should there be a `tsconfig.json` for the root specifically? Maybe packages could inherit from it?

Reply:
Or if the doc is expected to only exist for `opentelemetry-types`, maybe it would make sense to move the command there and simply do `lerna run docs` here.

Reply:
Makes sense, done.